src/components/VideoPlayer/index.js

```diff
@@ -452,6 +452,7 @@ export default class VideoPlayer extends PureComponent {
             data-video-id={videoId}
             muted={isMuted}
             controls={hasControls}
+            {...this.props}
```

Comment:
this will pass all props, but React doesn't like if you pass non-html attributes to html elements.  generally how i approach this is is by doing:

```
const {
  beforescreenComponent,
  pausescreenComponent,
  endscreenComponent,
  accountId,
  playerId,
  videoId,
  hasControls,
  isMuted,
  ...restProps
} = this.props


<video {...restProps} />
```

Reply:
note that you'll need to reference every expected prop you want to pull out of `restProps` in order to not pass invalid attributes

Reply:
@jstnjns you got it